pkg/eval/mods/path/path.go

```diff
@@ -176,3 +181,100 @@ func isRegular(path string) bool {
 	fi, err := os.Lstat(path)
 	return err == nil && fi.Mode().IsRegular()
 }
+
+//elvdoc:fn temp-dir
+//
+// ```elvish
+// temp-dir [&dir=$dir] [$pattern]
+// ```
+//
+// Create a temporary, unique, directory and output its name. The default for the `&dir` option is a
+// value appropriate for your system. You can use `&dir=/some/path` to specify a particular
```

Comment:
> The default for the `&dir` option is a value appropriate for your system.

Better to first say what the `&dir` option **is** before saying what it defaults to. Something like:

> The `&dir` option determines where the directory will be created, and its default value ...

Reply:
Done